src/components/search/AnalysesSearchResults.js

```diff
@@ -0,0 +1,128 @@
+import { useTranslation } from "i18n";
+
+export default function AnalysesSearchResults(props) {
+    const { searchTerm, updateResultCount } = props;
+    const [analysesSearchKey, setAnalysesSearchKey] = useState(
+        ANALYSES_SEARCH_QUERY_KEY
+    );
+    const [
+        analysesSearchQueryEnabled,
+        setAnalysesSearchQueryEnabled,
+    ] = useState(false);
+    const { t } = useTranslation(["analyses"]);
+    //TODO: pass `t` into this function
+    const analysisRecordFields = analysisFields(t);
+    const {
+        status,
+        data,
+        isFetchingMore,
+        fetchMore,
+        canFetchMore,
+        error,
+    } = useInfiniteQuery(analysesSearchKey, searchAnalysesInfinite, {
+        enabled: analysesSearchQueryEnabled,
+        getFetchMore: (lastGroup, allGroups) => {
+            const totalPage = Math.ceil(lastGroup?.total / 100);
+            if (allGroups.length < totalPage) {
+                return allGroups.length;
+            } else {
+                return false;
+            }
+        },
+    });
+
+    const loadMoreButtonRef = React.useRef();
+
+    useEffect(() => {
+        if (searchTerm && searchTerm.length > 2) {
+            setAnalysesSearchKey([
+                ANALYSES_SEARCH_QUERY_KEY,
+                {
+                    rowsPerPage: 100,
+                    orderBy: analysisRecordFields.NAME.key,
+                    order: constants.SORT_ASCENDING,
+                    filter: getAnalysesSearchQueryFilter(searchTerm, t),
+                },
+            ]);
+            setAnalysesSearchQueryEnabled(true);
+        } else {
+            setAnalysesSearchQueryEnabled(false);
+        }
+    }, [analysisRecordFields.NAME.key, searchTerm, t]);
+
+    const columns = React.useMemo(
+        () => [
+            {
+                Header: analysisRecordFields.NAME.fieldName,
+                accessor: analysisRecordFields.NAME.key,
+            },
+            {
+                Header: analysisRecordFields.STATUS.fieldName,
+                accessor: analysisRecordFields.STATUS.key,
+            },
+        ],
+        [
+            analysisRecordFields.NAME.fieldName,
+            analysisRecordFields.NAME.key,
+            analysisRecordFields.STATUS.fieldName,
+            analysisRecordFields.STATUS.key,
+        ]
+    );
+    if (status === "loading") {
+        return (
+            <TableLoading
+                numColumns={5}
+                numRows={100}
+                baseId="analysesSearchResults"
+            />
+        );
+    }
+    if (error) {
+        return <Typography> Unable to search </Typography>;
+    }
+    if (!data) {
+        return <Typography> No Analyses. </Typography>;
+    }
+    let flatdata = [];
+    data.forEach((page) => {
+        flatdata = [...flatdata, ...page.analyses];
+    });
+
+    if (status === "success") {
+        updateResultCount(data[0].total);
+    }
+
+    return (
+        <SearchResultsTable
```

Comment:
Just picking a random spot - at a glance, these `/search/*SearchResults` files look not too different from the `/search/detailed/*SearchResults` files.  Can either be reused for the other with a bit of refactoring?

Reply:
It looks like this is an rebasing issue. I refactored the code and moved all the `/search/*SearchResults` to `/search/detailed/*SearchResults`. so `/search/*SearchResults` files can deleted. May be I missed it during rebase 🙅

Reply:
This should be fixed now.